packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -316,13 +316,23 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
             color: game.backgroundColor(),
             child: LayoutBuilder(
               builder: (_, BoxConstraints constraints) {
-                game.onGameResize(constraints.biggest.toVector2());
+                final size = constraints.biggest.toVector2();
+                if (size.isZero()) {
+                  return widget.loadingBuilder?.call(context) ?? Container();
+                }
+                game.onGameResize(size);
                 return FutureBuilder(
                   future: loaderFuture,
                   builder: (_, snapshot) {
                     if (snapshot.hasError) {
                       final errorBuilder = widget.errorBuilder;
                       if (errorBuilder == null) {
+                        // @Since('2.16')
+                        // throw Error.throwWithStackTrace(
+                        //   snapshot.error!,
+                        //   snapshot.stackTrace,
+                        // )
+                        print(snapshot.stackTrace);
```

Comment:
Lets use at least the log method from the developer package here

Reply:
done